src/main/groovy/com/ceilfors/transform/gq/TimestampCodeFlowPrinter.groovy

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016 Wisen Tanasa
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ceilfors.transform.gq
+
+/**
+ * @author ceilfors
+ */
+class TimestampCodeFlowPrinter implements CodeFlowPrinter {
+
+    private CodeFlowPrinter out
+    private long startMilli
+    private Clock clock
```

Comment:
Always torn apart by this kind of problem. For example all the implementation of CodeFlowPrinters don't actually have purpose outside of CodeFlowManager?